build/templates/errors.py.mako

```diff
@@ -70,14 +70,14 @@ class DriverNotInstalledError(Error):
 
 
 class DriverTooOldError(Error):
-    '''An error due to using this module with an older version of the driver runtime.'''
+    '''An error due to using this module with an older version of the ${driver_name} driver runtime.'''
 
     def __init__(self):
         super(DriverTooOldError, self).__init__('A function was not found in the ${driver_name} runtime. Please visit http://www.ni.com/downloads/drivers/ to download a newer version and install it.')
 
 
 class DriverTooNewError(Error):
-    '''An error due to the driver runtime being too new for the Python module.'''
+    '''An error due to the ${driver_name} driver runtime being too new for this module.'''
 
     def __init__(self, prefix_message):
```

Comment:
This `prefix_message` requires client to know implementation details of the class. Specifically to leave a space after the period of the message, and what the appended string is in order to ensure it all makes sense.

I don't think this is worth it.

Just say:

> The ${driver_name} runtime returned an unexpected value. This can occur if it is too new for the ${module_name} Python module. Upgrade the ${module_name} Python module.

Reply:
I have removed `prefix_message` and updated the error message based on your suggestion in my latest commit.